manifests/satellite6.pp

```diff
@@ -0,0 +1,19 @@
+# Class: satellite6
+#   Install the SAtellite 6 flavor of the robottelo_slave
+#
+class robottelo_slave::satellite6 {
```

Comment:
I think "robottelo_slave::satellite6" is quite specific (I was thinking "robottelo_slave" could also be used for Katello). The main actual difference here is adding the 'brewkoji' packag. The existing class could already theoretically be a builder for upstream, this is why I tried to choose a name that would tell you that this is for downstream-only use.
If this entire repo is exclusively for downstream use, I guess I could just drop my new class and go and change the existing one.